### PR TITLE
Thanos compactor

### PR DESCRIPTION
**Pull request by thip**
Adds [thanos compactor](https://github.com/thanos-io/thanos/blob/main/docs/components/compact.md) to our ECS cluster. This ensures that long term metric data is stored efficiently in s3.

### CODE REVIEW

**Review by wanieldilson — COMMENTED** [review]
Do we need to increase the size of the ECS cluster to allow for this new task?

**Comment by thip** [comment]
> Do we need to increase the size of the ECS cluster to allow for this new task?

Nope, I seperated it out from the other prometheus stuff so it's got its own dedicated resources

File: mojo.tf [hunk 1]
```diff
@@ -84,6 +84,8 @@ module "grafana_v2" {
 module "prometheus_v2" {
   source = "./modules/prometheus"
 
+  enable_compactor = true
```
Comment (elcorbs) [inline mojo.tf:87]: You're using a mix of string and boolean, I don't think it'll make a difference but it's nice to be consistent

**Review by elcorbs — APPROVED** [review]

File: mojo.tf (hunk 1 above)
Comment (thip) [reply mojo.tf:87]: oops yes, good spot

Comment (thip) [reply mojo.tf]: relevent doc incase you're interested https://www.terraform.io/docs/configuration-0-11/variables.html#booleans

**Pull request merged**